src/Request.h

```diff
@@ -49,6 +50,13 @@ struct Request final
      */
     Request();
 
+    /**
+     * @brief Constructor
+     *
+     * Constructs Request with provided arguments
+     */
+    Request(const asn1SccPID sender_pid, const uint8_t* data, size_t length);
```

Comment:
length could be also const, there could probably be one more const in data (?)

Reply:
Done